### PR TITLE
mrexwhoami2/dev-1-add-content-wrappers

### PR DESCRIPTION
**Pull request by Afanasi0101**

### CODE REVIEW

File: src/components/header.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+function headerWrapper() {
+  const header = document.createElement('div');
+  header.classList.add('header-Wrapper');
```
Comment (aleksanderwalczuk) [inline src/components/header.js:3]: this `-Wrapper` shouldn't be capitalised.

File: src/components/contentWrapper.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+/* eslint-disable max-len */
+function createContentWrapper() {
+  const contentWrapper = document.createElement('div');
+  contentWrapper.classList.add('content-wrapper');
+  contentWrapper.dataset.view = 'main';
```
Comment (aleksanderwalczuk) [inline src/components/contentWrapper.js:5]: please remove the dataset, we won't use it for now.

File: src/components/contentWrapper.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
+/* eslint-disable max-len */
```
Comment (aleksanderwalczuk) [inline src/components/contentWrapper.js:1]: you shouldn't disable the `max-len` rule. Remove this comment.

File: src/scss/components/index.scss [hunk 4]
```diff
@@ -1,3 +1,5 @@
 // import components directory style files here
 @import "./example";
 @import "./btn.scss";
+@import "./contentWrapper.scss";
+@import "./header.scss";
```
Comment (aleksanderwalczuk) [inline src/scss/components/index.scss:5]: we don't need empty files. Please remove it.

**Review by aleksanderwalczuk — CHANGES_REQUESTED** [review]

File: src/components/contentWrapper.js [hunk 5]
```diff
@@ -0,0 +1,11 @@
+import createBtn from './btn';
+
+function createContentWrapper() {
+  const contentWrapper = document.createElement('div');
+  contentWrapper.classList.add('content-wrapper');
+  const btn = createBtn('play the game');
+  contentWrapper.appendChild(btn);
+  return contentWrapper;
```
Comment (aleksanderwalczuk) [inline src/components/contentWrapper.js:8]: @Afanasi0101 we should add the button to the contentWrapper like this.

File: src/app.js [hunk 6]
```diff
@@ -1,9 +1,12 @@
...
+import createContentWrapper from './components/contentWrapper';
 
 function app() {
   const root = document.querySelector('#app');
-  const btn = createBtn('play the game');
-  root.appendChild(btn);
+  const head = headerWrapper();
+  root.appendChild(head);
+  const contentBox = createContentWrapper();
+  root.appendChild(contentBox);
```
Comment (aleksanderwalczuk) [inline src/app.js:9]: So we can remove it from app ;)

**Review by aleksanderwalczuk — APPROVED** [review]
Please see final changes and merge when you do.

File: src/app.js (hunk 6 above)
Comment (Afanasi0101) [reply src/app.js:9]: why would we delete it?

**Pull request merged**